render/wlr_renderer.c

```diff
@@ -251,22 +251,12 @@ bool wlr_renderer_init_wl_display(struct wlr_renderer *r,
 }
 
 struct wlr_renderer *wlr_renderer_autocreate_with_drm_fd(int drm_fd) {
-	struct gbm_device *gbm_device = gbm_create_device(drm_fd);
-	if (!gbm_device) {
-		wlr_log(WLR_ERROR, "Failed to create GBM device");
-		return NULL;
-	}
-
-	struct wlr_egl *egl = wlr_egl_create(EGL_PLATFORM_GBM_KHR, gbm_device,
-			NULL);
+	struct wlr_egl *egl = wlr_egl_create_from_drm_fd(drm_fd);
 	if (egl == NULL) {
 		wlr_log(WLR_ERROR, "Could not initialize EGL");
-		gbm_device_destroy(gbm_device);
 		return NULL;
 	}
 
-	egl->gbm_device = gbm_device;
```

Comment:
This field is now unused, I think we can get rid of it (and also get rid of the `gbm.h` include).

Reply:
Done :D